src/open_inwoner/utils/ckeditor.py

```diff
@@ -5,12 +5,12 @@
 from bs4 import BeautifulSoup
 
 CLASS_ADDERS = [
-    ("h1", "h1"),
-    ("h2", "h2"),
-    ("h3", "h3"),
-    ("h4", "h4"),
-    ("h5", "h5"),
-    ("h6", "h6"),
+    ("h1", "utrecht-heading-1"),
+    ("h2", "utrecht-heading-2"),
+    ("h3", "utrecht-heading-3"),
+    ("h4", "utrecht-heading-4"),
+    ("h5", "utrecht-heading-5"),
+    ("h6", "utrecht-heading-6"),
```

Comment:
How much do we have in the database with the not-so-great h1,h2,h3 classes? 
I could change these into `("h2", "h2 utrecht-heading-2")` so it still contains the old class, but we need a way to override those styles so municipalities can, in future, change these styles with their own design-tokens.
Things to know before changing:  'utrecht' is not very specific; it has general values that can be overridden/expanded by the final styling of an OIP design-token, or a  municipality design-token.
The styling of the headings with the old .h1/.h2/.h3classes currently still depends on the H1/H2/H3 SCSS files where those styles are still defined, but we also wanted to get rid of those, so... what should we do?

Reply:
Could be quite a lot of product pages and everything else that is user editable.

Reply:
@Bartvaderkin Shall we then just create a separate task for migration if we ever wish to tackle that? 
For backward compatibility in this PR we actually still have the SCSS styling of the .h1/.h2/.h3... headings so nothing is changing for old templates actually, or please do tell how to go forward because this PR needs to do useful stuff.

Reply:
@alextreme this we could tak about on Monday.

Reply:
OK, @Bartvaderkin and @pi-sigma, in that case these changes in this 1 file I can reverse. And I can just add the utrecht **values** to the 'legacy' SCSS instead, without changing the HTML in productpages.

For post-processing: I created a new task for this, because there is no styling yet for the Utrecht Paragraph and Button components, which would be premature.

New issue here: https://taiga.maykinmedia.nl/project/open-inwoner/task/2465